src/components/SignerPanel/SignerPanel.js

```diff
@@ -174,16 +242,18 @@ class SignerPanel extends React.Component {
       status,
     } = this.state
 
+    const isTxSignReq = isTxSignerStatus(status)
```

Comment:
It's not so obvious what this is. Given that there are two modes, namely
- message signature
- transaction

What do you think about naming it simply `isTransaction` Signing request  is already implied by the component name an.

Alternatively, breaking the status types from:
`STATUS_CONFIRMING_TX,
  STATUS_SIGNING_TX,
  STATUS_SIGNED_TX,
  STATUS_ERROR_TX,
  STATUS_CONFIRMING_MSG_SIGN,
  STATUS_SIGNING_MESSAGE,
  STATUS_MESSAGE_SIGNED,
  STATUS_ERROR_SIGNING_MSG,
`

to two separate data types, e.g.`mode` and `status` might reduce the complexity given that both signatures and transactions follow the same state machine model.

Reply:
I like `isTransaction`! Latest commit has that change.

Would you also like me to implement the `mode` `status` logic instead of the 4 separate statuses?